### PR TITLE
Update persistence.md

### PR DESCRIPTION
**Pull request by 4everlandorg**
4EVERLAND is a Web 3.0 cloud computing platform that integrates storage, computing, and network core capabilities. Over 30,000 users and developers are using our decentralised service. More projects are using 4EVERLAND's services https://docs.4everland.org/more/use-cases. Strongly recommended that we should be added to this document. Please contact us from contact@4everland.org.

### CODE REVIEW

**Comment by ElPaisano** [comment]
@4everlandorg this looks great. To keep the list in alphabetical order, can you please move the link to the top of list?

**Comment by ElPaisano** [comment]
@2color what's your opinion on including this here?

**Comment by 2color** [comment]
As far as I know 4everland does not create storage deals on Filecoin. So I would only add it to the pinning services.

File: docs/concepts/persistence.md [hunk 1]
```diff
@@ -104,3 +105,4 @@ Options in this category include:
 - [Powergate](https://docs.filecoin.io/build/powergate)
 - [ChainSafe Storage](https://storage.chainsafe.io)
 - [Fleek Storage](https://fleek.co/storage)
+- [4EVERLAND Bucket](https://www.4everland.org/bucket)
```
Comment (2color) [inline docs/concepts/persistence.md:108]:
```suggestion
```

**Comment by 4everlandorg** [comment]
Hi, just curious why the changes I submitted were not approved by the bots?

**Comment by TMoMoreau** [comment]
@4everlandorg For some reason the automated link checker seems to be timing out on the estuary.tech link, not sure why. The link still works, and the checker does not need to pass to merge, so we're all good.

**Review by TMoMoreau — APPROVED** [review]
Looks good

**Comment by 4everlandorg** [comment]
Hi, is there any progress? Do we just have to wait for the Docs page to update?

**Pull request merged**